src/platform/silabs/rs911x/wfx_sl_ble_init.h

```diff
@@ -71,6 +71,17 @@ extern "C" {
 #define RSI_BLE_CHARACTERISTIC_TX_MEASUREMENT_HANDLE_LOCATION (4)
 #define RSI_BLE_CHARACTERISTIC_TX_GATT_SERVER_CLIENT_HANDLE_LOCATION (5)
 
+#ifdef CHIP_ENABLE_ADDITIONAL_DATA_ADVERTISING
+#define RSI_BLE_CUSTOM_CHARACTERISTIC_C3_SIZE (16)
```

Comment:
what does this size of 16 represent? Is this the data length C3 should 512.

Reply:
It is Size of UUID in bytes ,here 128-bit (16-byte) UUID.

Reply:
Please add this info with a relevant define name or comment
`RSI_BLE_CUSTOM_CHARACTERISTIC_C3_UUID_SIZE` would be better
but i'd also say `RSI_BLE_CUSTOM_CHARACTERISTIC` is a long prefix

Reply:
Modified the prefix to `RSI_BLE_CHAR_C3_` and pushed the changes